gef.py

```diff
@@ -7541,6 +7541,9 @@ def context_stack(self):
 
         return
 
+    def has_breakpoint_at_addr(self, address):
```

Comment:
This is inconsistent with the other `has_breakpoint`, since this one calls `gdb.breakpoints` and the other one is provided the list. Let's choose one style and stick with it.

nit: Maybe `addr_has_breakpoint` is a better name

Reply:
Yeah, I agree, but the reason why the other has a list as a parameter is because it is possible to filter all the breakpoints earlier by file name, and avoid scanning the unrelated one every time (In the context screen we see only one file at time) 
If the number of breakpoints is low, this is useless, but it might save some time with a lot of them in many different files. 

Maybe it is a pretty useless optimization, wdyt?

Reply:
it makes sense to filter outside. That's why I said choose one style. We can make this one accept a list of breakpoints, so it's consistent and we can still 'prefilter'.

We can also always change this, that's why we work in the dev branch :)

Reply:
Done it (: